homeassistant/components/ps4/const.py

```diff
@@ -9,6 +9,9 @@
 if TYPE_CHECKING:
     from . import PS4Data
 
+PS4_DOCS_URL = "https://www.home-assistant.io/components/ps4/"
+PSN_TOKEN_URL = "https://andshref.github.io/PS-4-Waker-REST/psn_token.html"
```

Comment:
The `PSN_TOKEN_URL` link doesn't appear to exist? I think this should also point to the existing HA docs.